azext_iot/_params.py

```diff
@@ -1120,6 +1126,12 @@ def load_arguments(self, _):
             options_list=["--device-information", "--info"],
             help="Optional device information.",
         )
+        context.argument(
+            "credential_policy_name",
+            options_list=["--credential-policy-name", "--credential-policy"],
```

Comment:
nit, for a pass of params to ensure concise options

Reply:
Ack - due to the CLI auto-resolving args, `--credential-policy` should also resolve to `--credential-policy-name`, so I think just leaving one full name and one short (like `--cpn`) saves us from updating docs but maintains at least a long/short option. Thoughts?
```suggestion
            options_list=["--credential-policy-name", "--cpn"],
```